$OpenBSD$

Set proper library dir
Use default compiler set by ports

Index: Makefile
--- Makefile.orig
+++ Makefile
@@ -1,7 +1,8 @@
 LIBTOOL = libtool
-LIBDIR = /usr/lib
-CXX = g++
-CXXFLAGS = -g -O3 -Wall -Wextra -Wshadow -Wconversion -Wcast-qual -Wformat=2
+INCLUDEDIR = ${PREFIX}/include/pystring
+LIBDIR = ${PREFIX}/lib
+CXX ?= g++
+CXXFLAGS = -g ${CFLAGS} -Wextra -Wshadow -Wconversion -Wcast-qual -Wformat=2
 
 all: libpystring.la
 
@@ -12,7 +13,8 @@ libpystring.la: pystring.lo
 	$(LIBTOOL) --mode=link --tag=CXX $(CXX) -o $@ $< -rpath $(LIBDIR)
 
 install: libpystring.la
-	$(LIBTOOL) --mode=install install -c $< $(LIBDIR)/$<
+	$(LIBTOOL) --mode=install install -Dm755 $< $(LIBDIR)/$<
+	$(LIBTOOL) --mode=install install -Dm644 pystring.h $(INCLUDEDIR)/pystring.h
 
 clean:
 	$(RM) -fr pystring.lo pystring.o libpystring.la .libs
